deployment/services.env.template

```diff
@@ -0,0 +1,15 @@
+PFS_HOST=0.0.0.0
+PFS_KEYSTORE_FILE=/keystore/<FILENAME>
```

Comment:
Is this meant to be filled in manually? If it is, adding a comment like
```
# Make a copy called `services.env` and replace all <PLACEHOLDERS>.
```
would make that more clear. Otherwise I would wonder if some tooling is supposed to do that.

Reply:
Good point